src/util/types.ts

```diff
@@ -92,22 +92,11 @@ export interface ChartCoordinate extends Coordinate {
   outerRadius?: number;
 }
 
-export type ScaleType =
-  | 'auto'
-  | 'linear'
-  | 'pow'
-  | 'sqrt'
-  | 'log'
-  | 'identity'
-  | 'time'
-  | 'band'
-  | 'point'
-  | 'ordinal'
-  | 'quantile'
-  | 'quantize'
-  | 'utc'
-  | 'sequential'
-  | 'threshold';
+type D3ScaleType = {
```

Comment:
so really all we gain here is `scaleSymlog` which we could add to the list manually

The other scales added are 
- `radial` (which is confusing because its a variation of a linear scale)
- sequential variants of scales https://d3js.org/d3-scale/sequential
- diverging variants of scales https://d3js.org/d3-scale/diverging
- `implicit` - which results in an error because it isn't a function and also isn't useful to us https://d3js.org/d3-scale/ordinal#scaleImplicit

We risk adding more breaky scale types when we define it like this.

Can we just add `symlog`? At the very least we need to `Exclude` `implicit`